README.md

```diff
@@ -1,2 +1,171 @@
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Raw postgresql queries]**
+- **[Query Results]**
+- **[Schema]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+-postgres installed
+-Laptop
+-Internet
+-Github
+-Command Line
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Samuelwanza/clinic-datanase.git
+```
+
+### Install
+
+Install this project with:
+
+Install postgreSQL [https://www.postgresql.org/download/]
+
+### Run tests
+
+To run tests,
+-copy the code in schema.sql in your psql commandline and execute
+-Copy the code in data.sql and run it in the psql commandline
+-Then run each of the queries in queries.sql one by one on the psql commandline
+
+### LIVE DEMO <a name="live-demo"></a>
+
+coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Samuel Wanza**
+
+- GitHub: [@Samuelwanza](https://github.com/Samuelwanza)
+- Twitter: [@Samuelwanza](https://twitter.com/samuelmunguti9)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/samuel-munguti-8a257a1bb/)
+
+**Cephas Tokoli**
+
+- GitHub: [@Ctokoli](https://github.com/ctokoli/)
+- Twitter: [@Ctokoli](https://twitter.com/ctokoli)
+- LinkedIn: [Cephas Tokoli](https://www.linkedin.com/in/ctokoli)
+
+**Majd Alkilany**
+
+- GitHub: [@majdalkilany](https://github.com/majdalkilany/)
+- LinkedIn: [majd alkilany](https://www.linkedin.com/in/majdalkilany)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Server integration]**
+- [ ] **[Cluster implementaion]**
+- [ ] **[Performance review]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse community.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] @Team kindly consider ensuring this MIT  link is working, It's not good to have a broken link in our readme file, its makes it unprofessional.

Reply:
- [ ] As mentioned in the previous review, the license link is broken. It will be nice to make sure there is no broken link in the README file to make it look more professional.